### DISCUSSION TITLE
integrate github teams into azure devops

### DISCUSSION THREAD

**Discussion by kf-raghu** [community/community#40456] [New to GitHub]
is there a way to import/export github teams into azure devops

**Comment by ossanna16** [comment]
Hi there @kf-raghu and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
